src/renderer/modals/Receive/steps/StepReceiveFunds.js

```diff
@@ -248,8 +262,16 @@ const StepReceiveFunds = ({
               address={address}
               showQRCodeModal={showQRCodeModal}
             />
-            <Separator />
-            <Receive2Device device={device} onVerify={onVerify} name={name} />
+            { isHederaAddress ? (
```

Comment:
it would be best to move that under families and create another specific UI there

Reply:
@LFBarreto Can you help me understand how the specific UI under the families folder takes precedence over Live Desktop's default UI?

Basically I would like to know how the specific UI gets injected and I think it would help me tremendously if you can help me understand, thank you very much in advance 🙏🏻

Reply:
Thank you @haammar-ledger! I'll get to this as soon as I can